### PR TITLE
[doc](agg_state) add agg_state in data-model

### PR DESCRIPTION
**Pull request by Mryange**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Mryange** [comment]
run buildall

File: docs/zh-CN/docs/data-table/data-model.md [hunk 1]
```diff
@@ -90,13 +90,16 @@ PROPERTIES (
 
 表中的列按照是否设置了 `AggregationType`，分为 Key (维度列) 和 Value（指标列）。没有设置 `AggregationType` 的，如 `user_id`、`date`、`age` ... 等称为 **Key**，而设置了 `AggregationType` 的称为 **Value**。
 
-当我们导入数据时，对于 Key 列相同的行会聚合成一行，而 Value 列会按照设置的 `AggregationType` 进行聚合。 `AggregationType` 目前有以下四种聚合方式：
+当我们导入数据时，对于 Key 列相同的行会聚合成一行，而 Value 列会按照设置的 `AggregationType` 进行聚合。 `AggregationType` 目前有以下四种聚合方式和agg_state：
```
Comment (BiteTheDDDDt) [inline docs/zh-CN/docs/data-table/data-model.md:93]: There are some other aggregation types, such as bitmap_union/hll_union...

**Review by BiteTheDDDDt — COMMENTED** [review]
need update engish doc also

**Comment by Mryange** [comment]
run buildall

**Review by BiteTheDDDDt — APPROVED** [review]

**Review by zclllyybb — APPROVED** [review]
LGTM

**Pull request merged**